sops.go

```diff
@@ -468,3 +468,109 @@ func ToBytes(in interface{}) ([]byte, error) {
 		return nil, fmt.Errorf("Could not convert unknown type %T to bytes", in)
 	}
 }
+
+// MapToMetadata tries to convert a map[string]interface{} obtained from an encrypted file into a Metadata struct.
+func MapToMetadata(data map[string]interface{}) (Metadata, error) {
+	var metadata Metadata
+	metadata.MessageAuthenticationCode, _ = data["mac"].(string)
```

Comment:
Should there be a friendly log message here to indicate that the MAC is ignored because none was found?